libvips/colour/XYZ2CMYK.c

```diff
@@ -2,7 +2,7 @@
  * vips_icc_export.
  *
  * 21/12/18
- *      - from CMYK2XYZ.c
+ *      - from CMYK2CMYK.c
```

Comment:
Search-and-replace typo I guess.

Reply:
Good catch, I will fix it in next patch. Thx :)